core/types/transaction_test.go

```diff
@@ -507,12 +508,20 @@ func TestTransactionCoding(t *testing.T) {
 				},
 				AccessList: accesses,
 			}
+		case 5:
+			txdata = &SignedBlobTx{
+				Message: BlobTxMessage{
+					ChainID:    Uint256View(*uint256.NewInt(1)),
+					Nonce:      Uint64View(i),
+					AccessList: AccessListView(accesses),
```

Comment:
I would add non-nil values for MaxFeePerDataGas and BlobVersionedHashes here.

Reply:
Updated